src/DataTable/TableRow.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import TableCell from './TableCell';
+
+const TableRow = ({ row }) => (
+  <tr {...row.getRowProps({
+    className: classNames({
+      'pgn__table-row': true,
+      'is-selected': row.isSelected,
+    }),
+  })}
+  >
+    {row.cells.map(cell => <TableCell cell={cell} key={`${cell.column.Header}${row.id}`} />)}
+  </tr>
+);
+
+TableRow.propTypes = {
+  row: PropTypes.shape({
```

Comment:
What do you think about unwrapping row so that the row object is the TableRow props. Usage would look like:
`<TableRow {...row} key={row.id} />`

Reply:
Yeah, I think that's reasonable.

Reply:
I liked it so much I did the same for TableCell and TableHeaderCell. Thank God for unit tests.